src/reflect/scala/reflect/internal/Symbols.scala

```diff
@@ -193,6 +193,16 @@ trait Symbols extends api.Symbols { self: SymbolTable =>
 
   private[reflect] case class SymbolKind(accurate: String, sanitized: String, abbreviation: String)
 
+  protected def newStubSymbol(owner: Symbol,
+                              name: Name,
+                              missingMessage: String,
+                              isPackage: Boolean = false): Symbol = {
```

Comment:
`isPackage` is actually unused after recent refactorings, could you remove in a setup commit for this one?